manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Blog app manager."""
```

Comment:
It's a function. Use a verb in the imperative mood.
```suggestion
    """Invoke blog app manager."""
```